modules/oe_theme_paragraphs_contact/oe_theme_paragraphs_contact.install

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Install hooks for OpenEuropa Theme Paragraphs Contact module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_install().
+ *
+ * Sets contacts rendering to use the full view mode.
+ */
+function oe_theme_paragraphs_contact_install() {
+  $entity_view_display = EntityViewDisplay::load('paragraph.oe_contact.default');
```

Comment:
This only needs to run if we are not in the middle of a config sync. See other hook_installs for example.